src/components/RichTextEditor.tsx

```diff
@@ -145,26 +157,27 @@ export default function RichTextEditor({
               body :first-child { margin-top: 0; }
               a { color: ${theme.palette.primary.main}; }
             `,
-          ].join("\n"),
-          minHeight: 300,
-          menubar: false,
-          plugins: ["autoresize", "lists link image", "paste help", "code"],
-          statusbar: false,
-          toolbar:
-            "formatselect | bold italic forecolor | link | bullist numlist outdent indent | removeformat code | help",
-          body_id: id,
-        }}
-        value={value}
-        onEditorChange={onChange}
-        onFocus={() => {
-          setFocus(true);
-          if (onFocus) onFocus();
-        }}
-        onBlur={() => {
-          setFocus(false);
-          if (onBlur) onBlur();
-        }}
-      />
+            ].join("\n"),
+            minHeight: 300,
+            menubar: false,
+            plugins: ["autoresize", "lists link image", "paste help", "code"],
+            statusbar: false,
+            toolbar:
+              "formatselect | bold italic forecolor | link | bullist numlist outdent indent | removeformat code | help",
+            body_id: id,
+          }}
+          value={value}
+          onEditorChange={onChange}
+          onFocus={() => {
+            setFocus(true);
+            if (onFocus) onFocus();
+          }}
+          onBlur={() => {
+            setFocus(false);
+            if (onBlur) onBlur();
+          }}
+        />
+      </Suspense>
```

Comment:
Nothing changed here, only covered with <Suspense>

There isn't any default styling, so if a problem happens on async loading it crashes. Seems impossible but I wanted to be deterministic here.